modules/ROOT/pages/documentation-openapi.adoc

```diff
@@ -105,4 +105,5 @@ You can view the documentation by pointing a browser at the `/openapi/ui` endpoi
 
 == See also
 
-Ready to try out documenting APIs with Open Liberty for yourself? Check out the guide for link:https://openliberty.io/guides/microprofile-openapi.html[Documenting RESTful APIs].
+link:https://openliberty.io/guides/microprofile-openapi.html[Documenting RESTful APIs]
```

Comment:
change
link:https://openliberty.io/guides/microprofile-openapi.html[Documenting RESTful APIs]
to
link:/guides/microprofile-openapi.html[Guide: Documenting RESTful APIs]